submissions/evgeniy24/friends_app/app.js

```diff
@@ -0,0 +1,248 @@
+const REQUEST_URL = 'https://randomuser.me/api/?results=20',
+    HEADER = document.querySelector('.header-name'),
+    FRIENDS_LIST_WRAP = document.querySelector('.friendsList-wrap'),
+    MENU_WRAP = document.querySelector('.menu-wrap'),
+    SEARCH_FIELD = document.querySelector('#search'),
+    SORTING_MENU = document.querySelector('.sorting-menu'),
+    FILTER_MENU = document.querySelector('.filter-menu'),
+    MOBILE_MENU_BUTTON = document.querySelector('.button-container');
```

Comment:
Don’t overuse global variables, keep data that you are using closest as possible to the place where you using it. As an example, you have saved elements that were used only in one place. So remove redundant globals